### PR TITLE
Import project: suggest a config file (yaml)

### PR DESCRIPTION
**Pull request by humitos**
I tried to add the intermediate page where we suggest a common Sphinx YAML file for the user to copy and paste.

I found the pattern a little hard to follow. The base template didn't have all the blocks I needed, so I added some more but I don't feel comfortable with them.

I also had to create a block to overwrite the CSS classes of the main div, which sounds weird to me as well.

Besides, I'm not sure how to use the CSS classes to achieve what I need: syntax highlighting, smaller text, etc.

Related https://github.com/readthedocs/readthedocs.org/pull/10356
Closes https://github.com/readthedocs/ext-theme/issues/165

### CODE REVIEW

**Comment by agjohnson** [comment]
I'll add some notes here for some reference, but in pulling this down to help with some guidance, I already made a few changes that this PR could use. But still, I'll note what I did here and probably move to merge this early so my work doesn't have to stay in conflict for any longer.

The main thing that this PR needed is some element structure to help constrain the vertical height. The form is quite long with a big example YAML in the middle and it's not super clear that it's even a form until you scroll to the bottom. I could see a modal being better here overall, but I used a `scrolling segement`, which is a native FUI element that allows for scrolling inside the segment.

I also reused the element structure from some of our existing code blocks -- build detail page pseudo terminal, the integration HTTP request/response, etc. This provides a clear designation between the code example and the rest of the page.

I also dialed back on the `block` elements that were changed/removed, and kept the help topics list on the config form page.

image

Some future additions would be a dropdown selection or tabbed interface with "Sphinx", "Mkdocs", "See our docs for more".

**Review by agjohnson — COMMENTED** [review]
I pulled this one down to experiment and give a bit of feedback, but ended up coming up with a few fixes in the process. I put these up for review in #185.

I only wanted to poked around on the form column widths, but ended up experimenting a bit more here and in the process solved a few points I wanted to put in review.

I opened up that PR so we could review the differences, but most of the commentary is in this review here.

File: readthedocsext/theme/templates/projects/import_base.html [hunk 1]
```diff
@@ -6,9 +6,9 @@
 {% block title %}{% trans "Add Project" %}{% endblock %}
 
 {% block content %}
-  {% block project_add_content %}
     <div class="ui very padded centered stackable grid">
-      <div class="ui ten wide tablet six wide computer column">
+    {% block project_add_content %}
+      <div class="{% block project_add_css_classes %}ui ten wide tablet six wide computer column{% endblock project_add_css_classes %}">
```
[the remark this answers is not in the document]
Comment (agjohnson) [reply readthedocsext/theme/templates/projects/import_base.html:11]: Update: I did find a pattern that works well, and so control of the element classes here is no longer necessary.

File: readthedocsext/theme/templates/projects/import_base.html [hunk 2]
```diff
@@ -42,6 +45,7 @@ <h1 class="ui medium header">
           {% block wizard_actions %}
             <div>
               {% if wizard.steps.prev %}
+                {# TODO: the previous button should not validate the form #}
```
[the remark this answers is not in the document]
Comment (humitos) [reply readthedocsext/theme/templates/projects/import_base.html:48]: Yeah, probably. I think it was because of the checkbox probably.

File: readthedocsext/theme/templates/projects/import_config.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<p class="info">
+    {% blocktrans trimmed %}
+    Make sure your project has a <code>.readthedocs.yaml</code> at the root of your repository. This file is required by Read the Docs to be able to build your documentation. You can <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">read more about this in our documentation</a>.
+    {% endblocktrans %}
+</p>
+
+<p class="info">
+    Here you have an example for a common Sphinx project:
+
+    <pre class="ui padded">
```
[the remark this answers is not in the document]
Comment (agjohnson) [reply readthedocsext/theme/templates/projects/import_config.html:19]: I have talked about writing up doc pages for an introduction into authoring new UI, this would be good to point out.

All of the elements and their variations are documented in the documentation, but you need to know the element in order to look up the documentation. In this case its a `segment` element and the docs for `padded` are here: https://fomantic-ui.com/elements/segment.html#padded

I always use `<div class="ui [varations...] [element_name]">` to keep this more clear what element is being used.

I used:

- [scrolling](https://fomantic-ui.com/elements/segment.html#scrolling)
- [inverted](https://fomantic-ui.com/elements/segment.html#inverted)

And you can see the usage in all of the examples by clicking on the show source buttons:

image

[the remark this answers is not in the document]
Comment (agjohnson) [reply readthedocsext/theme/templates/projects/import_config.html:19]: I opened up https://github.com/readthedocs/ext-theme/issues/187 to cover some topics like these, I might want to do this sometime this week after we get a deploy out.

**Review by agjohnson — APPROVED** [review]
I've rebased everything here, hopefully without breaking anything :crossed_fingers:
